root/report/CatNoLooksLikeIsrc.js

```diff
@@ -0,0 +1,82 @@
+/*
+ * @flow strict-local
+ * Copyright (C) 2021 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import PaginatedResults from '../components/PaginatedResults';
+import loopParity from '../utility/loopParity';
+import EntityLink from '../static/scripts/common/components/EntityLink';
+import ArtistCreditLink
+  from '../static/scripts/common/components/ArtistCreditLink';
+
+import ReportLayout from './components/ReportLayout';
+import type {ReportDataT, ReportReleaseCatNoT} from './types';
+
+const CatNoLooksLikeIsrc = ({
+  canBeFiltered,
+  filtered,
+  generated,
+  items,
+  pager,
+}: ReportDataT<ReportReleaseCatNoT>): React.Element<typeof ReportLayout> => (
+  <ReportLayout
+    canBeFiltered={canBeFiltered}
+    description={l(
+      `This report shows releases which have catalog numbers that look
+       like ISRCs. Assigning ISRCs to releases is almost always wrong,
+       but still happens sometimes, especially for releases added to
+       MusicBrainz by an artist/label. But ISRCs are codes assigned
+       to recordings, and should be linked to the appropriate recording
+       instead. That said, do make sure this is not a legitimate catalog
+       number that just happens to look like an ISRC!`,
+    )}
```

Comment:
I suggest to link to the documentation page to help with learning what an ISRC looks like. Untested:

```suggestion
    description={exp.l(
      `This report shows releases which have catalog numbers that look
       like ISRCs. Assigning ISRCs to releases is almost always wrong,
       but still happens sometimes, especially for releases added to
       MusicBrainz by an artist/label. But ISRCs are codes assigned
       to recordings, and should be linked to the appropriate recording
       instead. That said, do make sure this is not a legitimate catalog
       number that just happens to look like an {doc_link|ISRC}!`,
      {doc_link: '/doc/ISRC'},
    )}
```